src/components/Piano.vue

```diff
@@ -128,6 +128,10 @@ export default {
     noteActive(note) {
       return pianoState[note] === true
     },
+    
+    setActive(note) {
```

Comment:
Just a minor thing, but this isn't a `setActive` method - it's a `toggleActive` method since if it's `true` it becomes `false` and vice versa. Can we rename it?